src/misc/SuggestionsCache.ts

```diff
@@ -2,10 +2,6 @@ export class SuggestionsCache<T> {
   private cache: { [hash: string]: Promise<T> } = {};
 
   getSuggestions(hash: string, suggestionsFetcher: () => Promise<T>): Promise<T | null> {
```

Comment:
Should probably remove `| null` here and the same condition in `clearSuggestion`